### PR TITLE
Material ui

### PR DESCRIPTION
**Pull request by Qairkhan**
Реализована функция вывода текста в консоль лог

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  
+    handleSubmit(event) {
+      console.log(this.state.value);
+      event.preventDefault();
+    }
+  
+    render() {
+      return (
+        <form onSubmit={this.handleSubmit}>
+          <label>
```
Comment (Dmitry-White) [inline src/App.js:29]: "label" tag here is redundant - it's included in TextField MaterialUI component.
See the TextField prop called "label".

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { TextField } from '@material-ui/core';
+
+import './App.css'
+
+
+
+class App extends React.Component {
+    constructor(props) {
```
Comment (Dmitry-White) [inline src/App.js:9]: This is the old way to do it - consider using class properties to make your code cleaner.
https://medium.com/front-end-weekly/class-properties-with-react-dbc9b33e901c

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { TextField } from '@material-ui/core';
+
+import './App.css'
+
```
Comment (Dmitry-White) [inline src/App.js:5]: Consider removing redundant empty lines.

File: src/index.js [hunk 4]
```diff
@@ -1,4 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import 'typeface-roboto';
```
Comment (Dmitry-White) [inline src/index.js:3]: Consider adding an empty line between groups of imports.

File: src/App.js (hunk 1 above)
Comment (Qairkhan) [reply src/App.js:29]: done

File: src/App.js (hunk 3 above)
Comment (Qairkhan) [reply src/App.js:5]: done

File: src/index.js (hunk 4 above)
Comment (Qairkhan) [reply src/index.js:3]: done

File: src/App.js (hunk 2 above)
Comment (Qairkhan) [reply src/App.js]: done

**Pull request merged**